gsk4/Gir.toml

```diff
@@ -115,25 +136,59 @@ generate_builder = true
     manual = true # TODO: fix me once (https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3744) is merged
 
 [[object]]
-name = "Gsk.Renderer"
+name = "Gsk.GLShaderNode"
 status = "generate"
-trait_name = "GskRendererExt"
-manual_traits = ["GskRendererExtManual"]
     [[object.function]]
-    name = "render"
-    # uses IsA<RenderNode> but RenderNode is not an Object
+    name = "new"
     manual = true
-    doc_trait_name = "GskRendererExtManual"
     [[object.function]]
-    name = "render_texture"
-    # uses IsA<RenderNode> but RenderNode is not an Object
+    name = "get_child"
+    manual = true # assert that idx < n_children
+
+[[object]]
+name = "Gsk.LinearGradientNode"
+status = "generate"
+    [[object.function]]
+    name = "new"
     manual = true
-    doc_trait_name = "GskRendererExtManual"
+
+[[object]]
+name = "Gsk.RadialGradientNode"
+status = "generate"
+    [[object.function]]
+    name = "new"
+    manual = true
```

Comment:
Why are all these manual?

Reply:
i.e. add comments here

Reply:
They will all be auto generated once the other gir PR is merged. Eg, see the other gtk4-rs PR